src/components/phase-banner/index.md.njk

```diff
@@ -15,7 +15,10 @@ Use the phase banner component to show users your service is still being worked
 
 ## When to use this component
 
-You must use the phase banner component if your service is still being developed and is hosted on the service.gov.uk subdomain.
+You must use the phase banner component if your service:
+- still needs to complete a live service assessment
+- is hosted on the service.gov.uk subdomain
```

Comment:
```suggestion
You must use the phase banner component if your service is hosted on the service.gov.uk subdomain and still needs to complete a live service assessment.
```

Reply:
Possibly a a mismatch between 'services' as a plural and 'it completes' being singular? Also not sure about 'completes' as it suggests the service is an active participant in the assessment process 😆 

I'd also personally use 'domain' rather than 'subdomain' in this context which seems to be consistent with the Service Manual (https://www.gov.uk/service-manual/technology/get-a-domain-name)

What about…

```suggestion
Services hosted on a service.gov.uk domain must use the phase banner until they pass a live assessment.
```

or…

```suggestion
Services hosted on a service.gov.uk domain must use the phase banner until they have passed a live assessment.
```